adapters/taboola/taboola.go

```diff
@@ -89,6 +106,36 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 	return bidResponse, errs
 }
 
+func (a *adapter) buildRequest(request *openrtb2.BidRequest, mediaType string) (*adapters.RequestData, error) {
```

Comment:
this function does not need to be defined as function receiver. 
```
func buildRequest(request *openrtb2.BidRequest, mediaType string) (*adapters.RequestData, error)
```

Reply:
I calls another method so it should receive the adapter and to be a method 
`a.buildEndpointUR`